src/App.vue

```diff
@@ -1,8 +1,8 @@
 <template>
   <v-app id="app">
       <Navbar></Navbar>
-
-      <v-container class="full-height-content">
+      <router-view v-if="customLayout"/>
```

Comment:
I guess this might be a sign in future we should move the "rows and cols" into each page. I suspect in most cases each "page" really wants control of it's layout rather than having it imposed on it at the App level